src/QsCompiler/Transformations/ClassicallyControlledTransformation.cs

```diff
@@ -9,21 +9,24 @@
 using Microsoft.Quantum.QsCompiler.SyntaxTokens;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
 using Microsoft.Quantum.QsCompiler.Transformations.Core;
+using Microsoft.Quantum.QsCompiler.Transformations.SearchAndReplace;
 
 
-namespace Microsoft.Quantum.QsCompiler.Transformations.ClassicallyControlledTransformation
+namespace Microsoft.Quantum.QsCompiler.Transformations.ClassicallyControlled
```

Comment:
I sort of would like to rename this file to ClassicallyControlled.cs as well, although in this case the Transformation part provides contrast with ClassicallyControlledUtils.cs.